src/Features/CSharp/Portable/Copilot/CSharpImplementNotImplementedExceptionFixProvider.cs

```diff
@@ -0,0 +1,169 @@
+            var methodOrProperty = (MemberDeclarationSyntax)syntaxNode;
+
+            SyntaxNode? replacement;
+            if (!memberImplementationDetails.TryGetValue(methodOrProperty, out var implementationDetails))
+            {
+                replacement = AddErrorComment(methodOrProperty);
+                Logger.Log(FunctionId.Copilot_Implement_NotImplementedException_Failed, logLevel: LogLevel.Error);
+            }
+            else
+            {
+                replacement = implementationDetails.ReplacementNode;
+                if (replacement != null && (replacement is BasePropertyDeclarationSyntax or BaseMethodDeclarationSyntax))
+                {
+                    replacement = replacement
+                        .WithLeadingTrivia(methodOrProperty.GetLeadingTrivia())
+                        .WithTrailingTrivia(methodOrProperty.GetTrailingTrivia())
+                        .WithAdditionalAnnotations(Formatter.Annotation, WarningAnnotation, Simplifier.Annotation);
+                }
+                else
+                {
+                    replacement = AddErrorComment(methodOrProperty, implementationDetails.Message);
+                    Logger.Log(FunctionId.Copilot_Implement_NotImplementedException_Failed, logLevel: LogLevel.Error);
+                }
+            }
+
+            editor.ReplaceNode(methodOrProperty, replacement);
+        }
+
+        var changedRoot = editor.GetChangedRoot();
+        var formattingOptions = await document.GetSyntaxFormattingOptionsAsync(cancellationToken).ConfigureAwait(false);
+        var spansToFormat = FormattingExtensions.GetAnnotatedSpans(changedRoot, new());
+        var formattedRoot = CSharpSyntaxFormatting.Instance.GetFormattingResult(changedRoot, spansToFormat, formattingOptions, new(), cancellationToken).GetFormattedRoot(cancellationToken);
+        changedRoot = formattedRoot;
+
+        editor.ReplaceNode(editor.OriginalRoot, changedRoot);
+        Logger.Log(FunctionId.Copilot_Implement_NotImplementedException_Completed, logLevel: LogLevel.Information);
+    }
+
+    private static Task<ImmutableArray<ReferencedSymbol>> FindReferencesAsync(Document document, ISymbol symbol, CancellationToken cancellationToken)
+    {
+        cancellationToken.ThrowIfCancellationRequested();
+        var searchOptions = FindReferencesSearchOptions.GetFeatureOptionsForStartingSymbol(symbol);
+        return SymbolFinder.FindReferencesAsync(symbol, document.Project.Solution, searchOptions, cancellationToken);
+    }
+
+    private static MemberDeclarationSyntax AddErrorComment(MemberDeclarationSyntax member, string? message = null)
+    {
+        var errorMessage = string.IsNullOrWhiteSpace(message) ? CSharpFeaturesResources.Failed_to_receive_implementation_from_Copilot_service : message;
+        var comment = SyntaxFactory.TriviaList(
+            SyntaxFactory.Comment($"/* {errorMessage} */"),
```

Comment:
```suggestion
            SyntaxFactory.Comment($"// {errorMessage}"),
```

Also, have tests where the error message is multiline.

Reply:
what's wrong with `/* */`

I like it better for multi line comments. I already have a multi line test

```
/* The generated implementation isn't a valid method or property:
using System;
class C
{
    void M()
    {
        throw new NotImplementedException();
    }
} */
```

Reply:
/**/  is njust not commonly used in our ecosystem.  but if you want to stick with that, it's fine :)